app/src/features/datasets/components/DatasetArtifacts.tsx

```diff
@@ -61,8 +61,13 @@ const DatasetAttachments: React.FC<IDatasetAttachmentsProps> = (props) => {
   const artifactsDataLoader = useDataLoader(() => biohubApi.dataset.getDatasetArtifacts(datasetId));
   artifactsDataLoader.load();
 
-  const artifactsList = artifactsDataLoader.data?.artifacts || [];
-  const numPendingDocuments = artifactsList.filter((artifact) => artifact.security_review_timestamp === null).length;
+  const artifactsList = artifactsDataLoader.data || [];
+
+  const numPendingDocuments =
+    artifactsList.filter(
+      (artifact) => artifact.supplementaryData.persecutionAndHarm === SECURITY_APPLIED_STATUS.PENDING
+    ).length || 0;
```

Comment:
You can remove this coelesce; `artifactsList` is guarenteed to be an array, so the length is guarenteed to be defined.